### DISCUSSION TITLE
Add current location to the "Overview" dashboard (Grafana)

### DISCUSSION THREAD

**Discussion by fatbasstard** [teslamate-org/teslamate#1148] [Show and tell]
I think it would make sense (well, I would like it) to add the current location of the car to the "Overview" dashboard.

Not sure if this a value that is present somewhere, but else it could be based on the end position of the last drive.

**Comment by fatbasstard** [comment]
Got an quick example: 

![image](https://user-images.githubusercontent.com/35613489/103285432-6f12c300-49de-11eb-9a4f-61c2b184d290.png)


If people think this is "useful" I can create a PR.

**Reply by jheredianet** [reply]
Checkout this [Teslamate Custom Grafana Dashboards](https://github.com/jheredianet/Teslamate-CustomGtrafanaDashboards) repository. 

![image](https://user-images.githubusercontent.com/45735191/212675476-049059ef-7d17-4fa2-96a4-1efbebf35d2a.png)

**Reply by bahree** [reply]
Nice. Is there a PR for this?

**Reply by jheredianet** [reply]
No, since I'll try to do it soon, I hope they accept the PR soon,  but in the mean time you could import them following the steps in the repo.

**Comment by DrMichael** [comment]
Looks great... Could even be a bit greater, not sure where though

While you're at it...

Could we change the interval for the temperature from 10m to perhaps 4h? That would still make sense and otherwise it is almost always "No data".

**Comment by DrMichael** [comment]
Didn't you post a newer solution? Are you going for a PR?

(I would rather reduce the height still. The states line is already not visible on 1024x780 resolution.)
